benchmarks/multimodal/jsonl/generate_videos.py

```
@@ -0,0 +1,146 @@
+) -> list[str]:
+    """Sample video slots from a fixed pool, no duplicates within each request.
+
+    Every video in the pool is guaranteed to appear at least once.
+    """
+    pool_size = len(pool)
+    total_slots = num_requests * videos_per_request
+    assert (
+        pool_size >= videos_per_request
+    ), f"videos-pool ({pool_size}) must be >= videos-per-request ({videos_per_request})"
+    assert total_slots >= pool_size, (
+        f"total slots ({num_requests}×{videos_per_request}={total_slots}) < "
```

Ruff (0.15.14)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)